echopype/calibrate/calibrate_azfp.py

```diff
@@ -112,30 +130,49 @@ def _cal_power(self, cal_type, **kwargs):
         See calc_Sv_offset() in convert/azfp.py
         """
         # Compute range in meters
-        self.compute_range_meter(cal_type=cal_type)  # range computation different for Sv and Sp per AZFP matlab code
+        self.compute_range_meter(
+            cal_type=cal_type
+        )  # range computation different for Sv and Sp per AZFP matlab code
 
         # Compute various params
-        spreading_loss = 20 * np.log10(self.range_meter)  # TODO: take care of dividing by zero encountered in log10
-        absorption_loss = 2 * self.env_params['sound_absorption'] * self.range_meter
-        SL = self.cal_params['TVR'] + 20 * np.log10(self.cal_params['VTX'])  # eq.(2)
-        a = self.cal_params['DS']  # scaling factor (slope) in Fig.G-1, units Volts/dB], see p.84
-        EL = self.cal_params['EL'] - 2.5 / a + self.echodata.beam.backscatter_r / (26214 * a)  # eq.(5)
-
-        if cal_type == 'Sv':
+        spreading_loss = 20 * np.log10(
+            self.range_meter
+        )  # TODO: take care of dividing by zero encountered in log10
```

Comment:
would be better to move the comment on top of the line and retain the code in one line